data/sles4sap/qe_sap_deployment/qesap_azure_sapconf_no_reg.yaml

```diff
@@ -34,16 +34,16 @@ ansible:
     primary_site: 'goofy'
     secondary_site: 'miky'
     use_sap_hana_sr_angi: '%USE_SR_ANGI%'
-  create:
-    - fully-patch-system.yaml
-    - pre-cluster.yaml
-    - sap-hana-preconfigure.yaml -e use_sapconf=true -e use_reboottimeout=900
-    - cluster_sbd_prep.yaml
-    - sap-hana-storage.yaml
-    - sap-hana-download-media.yaml
-    - sap-hana-install.yaml
-    - sap-hana-system-replication.yaml
-    - sap-hana-system-replication-hooks.yaml
-    - sap-hana-cluster.yaml
-  destroy:
+  sequences:
```

Comment:
remind me - is `sequences` optional? Can tests continue to use simple `create` etc sections without sequences in apiver 4?